src/main/java/Commands/ByeCommand.java

```diff
@@ -0,0 +1,27 @@
+package Commands;
```

Comment:
It is a better convention for package should be named in all lowercase. This is the same across some other package.